src/agents/definitions/log-analysis-summarizer.js

```
@@ -4,6 +4,7 @@ export default {
   name: "Log Analysis Summarizer",
   description:
     "Paste raw log output and get a structured summary of errors, warnings, patterns, timeline of events, and likely root cause — so you find the 5 lines that matter in 500 lines of noise.",
+  category: "DevOps",
```

Biome (2.4.16)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)